src/IO/ConcatSeekableReadBuffer.h

```diff
@@ -8,10 +8,10 @@ namespace DB
 {
 
 /// Reads from the concatenation of multiple SeekableReadBuffer's
-class ConcatSeekableReadBuffer : public SeekableReadBufferWithSize
+class ConcatSeekableReadBuffer : public SeekableReadBuffer
```

Comment:
```suggestion
class ConcatSeekableReadBuffer : public SeekableReadBuffer, public WithFileSize
```
and below `getTotalSize()` -> `getFileSize()`